config/configuration.toml.in

```diff
@@ -20,6 +20,24 @@ default_vcpus = -1
 #default_memory = @DEFMEMSZ@
 disable_block_device_use = @DEFDISABLEBLOCK@
 
+# Enable pre allocation of VM RAM, default false
+# Enabling this will result in lower container density
+# as all of the memory will be allocated and locked
+# This is useful when you want to reserve all the memory
+# upfront or in the cases where you want memory latencies
+# to be very predictable
+# Default false
+#enable_mem_prealloc = true
+#
+# Enable swap of vm memory. Default false.
+# The behaviour is undefined if mem_prealloc is also set to true
+#enable_swap = true
+#
+# Debug changes the default hypervisor and kernel parameters to
+# enable debug output where available.
```

Comment:
Can you put a comment explaining where that debug output goes / how to view it?

Reply:
Will do. I am also writing an Advanced debugging readme. I will add that to it too.

That will consist of
https://github.com/clearcontainers/runtime/issues/559
https://github.com/clearcontainers/runtime/issues/558

and anything else appropriate